packages/cli/lib/prompts/projectDevTargetAccountPrompt.js

```diff
@@ -109,13 +108,8 @@ const selectDeveloperTestTargetAccountPrompt = async (
     logger.debug('Unable to fetch developer test account usage limits: ', err);
   }
 
-  const devTestAccounts = accounts
-    .reverse()
-    .filter(
-      config =>
-        isDeveloperTestAccount(config) &&
-        config.parentAccountId === defaultAccountId
-    );
+  const accountIds = accounts.map(account => account.portalId);
```

Comment:
Nit: It seems like we are only using this in one `if` statement, so it could just be within that statement.